### ISSUE TITLE
Export Project Should Automatically export project to directory of the parent document

### ISSUE THREAD

**Issue by vivekkpatel** [rubberduck-vba/Rubberduck#5818]
**Justification**
Add Setting to always export project to directory of excel sheet so we don't have to select Folder every time we want to export Project

**Comment by daFreeMan** [comment]
If not duplicate of #5431, at least an extension to it.

**Comment by vivekkpatel** [comment]
Yes, Most User who like to export files for Git, likely to keep all modules in same folder

**Comment by retailcoder** [comment]
Thanks for the feedback, that's a great idea!
A simple "export source files to host document directory" checkbox (cleared by default, to preserve current "prompt everytime" default behavior) in *general settings* could do it.
The only thing to watch out for, I think, is that the setting isn't useful when hosted in VB6 and the notion of a "host document" would be confusing then: the option should probably be hidden when the host is VB6.

**Comment by mansellan** [comment]
VB6 needs separate thought. I'd love to be able to have Rubberduck encourage the use of actual file system folders, something which is entirely possible but never used because the Project Explorer doesn't show folders.

Maybe we should have a VB6-specific inspection, which would identify mismatches between the Folder annotation and the source file location, with a quickfix of "Move to folder" which would then move the source file and update the .vbp accordingly. If there's appetite for that I'll raise a separate issue.

**Comment by retailcoder** [comment]
@mansellan the quick-fix also needs to eventually actually delete the source code file from its original location. Definitely needs its own issue, but I think we must implement #4646 first.
